code/src/pages/MoviesList.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import './movielist.css'
+
+const POPLIST_API = "https://api.themoviedb.org/3/movie/popular?api_key=ab5cef50889fc06c1a1e75470eefe28b&language=en-US&page=1";
+
+export const MoviesList = () => {
+  const [moviesList, setMoviesList] = useState([]);  // stores moviesList
+
+
+  useEffect(() => {
+    fetch(POPLIST_API) 
+      .then(res => res.json())
+      .then(data => {
+        setMoviesList(data.results) //get list of movies frm api and pute data in state 
+      });
+  }, [])
+
+  return (
+//map over state to render chosen elements
+    <div className="movies_list">
+      {moviesList.map(({id, poster_path, title, release_date}) => (
+        <div key={id} className="movie_wrapper">
+          <Link key={moviesList.id} to={`movies/${id}`}>
+            <img className="movies_list_img" src={`https://image.tmdb.org/t/p/w342${poster_path}`} alt="movie poster" />  
```

Comment:
Here in `alt="movie poster" /> ` you could use `{title}` so the alt says more specific what poster it is and better for accessibility. So for example `alt="{title} movie poster" />` 😄

Reply:
Yes! Thanks :)